content/docs/reference/config/authorization/constraints & properties/index.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
would you not use spaces in filenames please?  I think it makes it harder on everyone to work on repos with spaces.

Cheers
-steve

Reply:
Done.